mapswipe_workers/tests/unittests/test_groups_overlap.py

```diff
@@ -0,0 +1,101 @@
+import os
+import unittest
+from mapswipe_workers.utils import tile_grouping_functions as t
+
+
+class TestGroupsOverlap(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = os.path.dirname(os.path.abspath(__file__))
+
+    def test_project_geometries_intersection(self):
+        zoom = 18
+        project_extent_file = os.path.join(
+            self.test_dir, "fixtures/completeness/closed_polygons.geojson"
+        )
+
+        groups_with_overlaps = t.extent_to_groups(project_extent_file, zoom, 100)
+        t.vertical_groups_as_geojson(
+            groups_with_overlaps, "groups_with_overlaps.geojson"
+        )
+
+        # remove groups within other groups
+        groups_with_overlaps_2 = t.remove_groups_within_other_groups(
+            groups_with_overlaps, zoom
+        )
+        t.vertical_groups_as_geojson(
+            groups_with_overlaps, "groups_with_overlaps2.geojson"
+        )
+
+        # TODO: remove once implemented in real function
+        groups, overlaps_total = t.adjust_overlapping_groups(
+            groups_with_overlaps_2, zoom
+        )
+        t.vertical_groups_as_geojson(groups, "groups.geojson")
+
+        groups_2, overlaps_total_2 = t.adjust_overlapping_groups(groups, zoom)
+        t.vertical_groups_as_geojson(groups_2, "groups_2.geojson")
+
+        # we expect 117 groups
+        self.assertEqual(len(groups_2), 117)
+
+    '''
```

Comment:
I guess we should remove this part here, since it's not used anyway and we can properly implement the tests once needed

Reply:
do you mean the whole test function or just the lines 38 and 39?

Reply:
I think we can the tests that are currently in comments are not used